templates/music/music.html

```diff
@@ -17,7 +17,7 @@
             {% for album in albums %}
                 <div class="column">
                     <div class="artist-list-image img-list">
-                        {% include "object_preview.html" with object=album img=album.artwork.img %}
+                        {% include "object_preview_music.html" with object=album img=album.artwork.img %}
```

Comment:
I extended `object_preview_music.html` from `object_preview.html` to keep most of the HTML we have in that. Let me know what you think of that change.